xbmc/AppParamParser.cpp

```diff
@@ -35,6 +35,8 @@
 #endif
 #include <stdlib.h>
 
+using namespace KODI::MESSAGING;
```

Comment:
A bit late to start arguing over namespace naming now, pr was open for a month so there was plenty of time for that